README.md

```diff
@@ -1,49 +1,55 @@
-# Catroweb
-
-[![Tests](https://github.com/Catrobat/Catroweb/actions/workflows/tests.yaml/badge.svg)](https://github.com/Catrobat/Catroweb/actions/workflows/tests.yaml)
-![Code Quality Checks](https://github.com/Catrobat/Catroweb-Symfony/workflows/Code%20Quality%20Checks/badge.svg)
-[![Dependabot Status](https://img.shields.io/badge/Dependabot-active-green.svg)](https://dependabot.com)
-[![codecov](https://codecov.io/gh/Catrobat/Catroweb/branch/develop/graph/badge.svg)](https://codecov.io/gh/Catrobat/Catroweb)
+⚠️ **__If you have not heard about who Catrobat is, what we are doing and in what ways you can contribute, then you might want to take a step back and check out the start of our [step-by-step guide](https://github.com/Catrobat/Catroid/blob/develop/README.md).__**
+```diff
+👷 TODO: Change link to README in Catrobat's .github repository when added!
+```
+💡 *If any step in the following process is unclear, misleading, or incorrect, you can directly contribute by issuing a pull request with the updated information!*
 
-[![Php 8.1](https://img.shields.io/badge/PHP-8.1-informational.svg)](https://www.php.net/)
-[![Symfony](https://img.shields.io/badge/Symfony-6.1-informational.svg)](https://symfony.com/)
 
-**Catroweb** is the name of the **Pocket Code share community platform** for the associated [Catrobat](https://github.com/Catrobat/) project. 
 
-The website features Catrobat projects (also called Catrobat programs) created with Pocket Code. 
-Uploaded projects can be shared and downloaded. Additionally, the platform provides users with features typical for a social network.
+# Catroweb
+Catroweb is the name of the "Pocket Code" [sharing platform](https://share.catrob.at/app/), where our community can upload their projects. If you uploaded your game in the previous step, it should already be visible to all other users. They can now download and remix your masterpiece. Additionally, the platform provides users with features typical for a social network.
```

Comment:
game is only a subset of the possible projects. Should we really reduce the set to games here?